webcrypto/subtle_crypto_test.go

```diff
@@ -167,6 +183,23 @@ func TestSubtleCryptoEncryptDecrypt(t *testing.T) {
 
 		assert.NoError(t, gotErr)
 	})
+
+	t.Run("RSA-OAEP", func(t *testing.T) {
+		t.Parallel()
+
+		ts := newWebPlatformTestRuntime(t)
+
+		gotErr := ts.EventLoop.Start(func() error {
+			err := executeTestScripts(ts.VU.Runtime(), webPlatformTestSuite+"encrypt_decrypt", "rsa_vectors.js", "rsa.js")
+			require.NoError(t, err)
+
+			_, err = ts.VU.Runtime().RunString(`run_test()`)
```

Comment:
nit: a lot of this test semm ... the same apart from the name and the some strings here.

It might be .. .better to make them into a table test?

Reply:
Sure, let me address this after merging this and https://github.com/grafana/xk6-webcrypto/pull/87 as part as clean up